src/cronjobs/userProfileJobs.js

```diff
@@ -19,8 +19,17 @@ const userProfileScheduledJobs = function () {
     'America/Los_Angeles',
   );
 
+  const updateUserStatusToActive = new CronJob(
+    '1 0 * * *', // Run this every day, 1 minute after mimdnight (PST).
+    userhelper.reActivateUser,
+    null,
+    false,
+    'America/Los_Angeles',
```

Comment:
Can we keep the time zone string somewhere as a generic constant so that any future zone changes would be a in a single place.

Reply:
Good point, we certainly could change it to a constant, however I don't think it's a big deal to keep those timezone strings as they are. I did discuss with Jae a few weeks ago the various possibilities of handling timezones and he wants it all in LA time (PST), so I don't think it will be changing.